### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by DannyLuu**
Updated the readme with the latest info.

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="app"/>
+	<classpathentry kind="src" path="target/scala-2.11/routes/main"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="output" path="target/scala-2.11/classes"/>
+</classpath>
```
Comment (wsmidt) [inline .classpath]: do not add this generated fileif you need to you can add .classpath to .gitignore and commit that

File: .project [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+			<name>org.scala-ide.sdt.core.scalabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.scala-ide.sdt.core.scalanature</nature>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+	</natures>
+</projectDescription>
```
Comment (wsmidt) [inline .project]: do not add this generated fileif you need to you can add .project to .gitignore and commit that

File: .classpath (hunk 1 above)
Comment (DannyLuu) [reply .classpath]: Done

File: .project (hunk 2 above)
Comment (DannyLuu) [reply .project]: Done?

**Review by wsmidt — APPROVED** [review]
:+1:

**Pull request merged**